libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/preview/CarRouteLine.kt

```diff
@@ -44,16 +48,17 @@ class CarRouteLine internal constructor(
             .build()
     }
 
-    private lateinit var routeLineView: MapboxRouteLineView
-    private lateinit var routeLineApi: MapboxRouteLineApi
-    private lateinit var routeArrowApi: MapboxRouteArrowApi
-    private lateinit var routeArrowView: MapboxRouteArrowView
+    private var routeLineView: MapboxRouteLineView? = null
+    private var routeLineApi: MapboxRouteLineApi? = null
+    private var routeArrowApi: MapboxRouteArrowApi? = null
+    private var routeArrowView: MapboxRouteArrowView? = null
```

Comment:
Can we restore `lateinit` since you moved the registration of observers back into the style listener?